### PR TITLE
Update image delivery serve chapter to allow custom domains

### PR DESCRIPTION
**Pull request by xjewer**
Add documentation for the custom domain image delivery.

### CODE REVIEW

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 1]
```diff
@@ -33,4 +33,27 @@ In this example:
...
+
+Here is aforementioned example, which the change of using custom domain:
+
+```txt
+https://somecustomdomain.com/cdn-cgi/imagedelivery/ZWd9g1K7eljCn_KDTu_MWA/083eb7b2-5392-4565-b69e-aff66acddd00/public
+```
+
+In this example, `:images_account_hash`, `:image_id` and `:variant_name` are the same, but the hostname and prefix path is different: 
+
+- `somecustomdomain.com` custom domain.
```
Comment (sejoker) [inline products/images/src/content/cloudflare-images/serve-images/index.md]: Cloudflare proxied domain under the same account as the Cloudflare Images.

**Review by sejoker — CHANGES_REQUESTED** [review]

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 2]
```diff
@@ -33,4 +33,27 @@ In this example:
 - `083eb7b2-5392-4565-b69e-aff66acddd00` is the image id.
 - `public` is the variant name.
 
-When a client requests an image, Cloudflare Images will pick the optimal format between WebP, PNG, JPEG and GIF. The format Cloudflare serves to the user is determined by client headers and the image type.
\ No newline at end of file
+When a client requests an image, Cloudflare Images will pick the optimal format between WebP, PNG, JPEG and GIF. The format Cloudflare serves to the user is determined by client headers and the image type.
+
+## Serving images from custom domains
+
+Image delivery is supported from all customer domains under the same Cloudflare account.
```
Comment (sejoker) [inline products/images/src/content/cloudflare-images/serve-images/index.md]: I would add:
Serving images from imagedelivery.net  requires additional TLS negotiations on the client-side, adding latency and impacting loading performance. That's why image delivery is supported from all customer domains under the same Cloudflare account...

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 3]
```diff
@@ -33,4 +33,27 @@ In this example:
...
+To serve images through custom domains, an image url should be adjusted to the following format:
+
+```txt
+https://<customerdomain.net>/cdn-cgi/imagedelivery/:images_account_hash/:image_id/:variant_name
```
Comment (sejoker) [inline products/images/src/content/cloudflare-images/serve-images/index.md]: s/<customerdomain.net>/somecustomdomain.com

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 4]
```diff
@@ -33,4 +33,27 @@ In this example:
...
 - `public` is the variant name.
 
-When a client requests an image, Cloudflare Images will pick the optimal format between WebP, PNG, JPEG and GIF. The format Cloudflare serves to the user is determined by client headers and the image type.
\ No newline at end of file
+When a client requests an image, Cloudflare Images will pick the optimal format between WebP, PNG, JPEG and GIF. The format Cloudflare serves to the user is determined by client headers and the image type.
+
+## Serving images from custom domains
+
+Image delivery is supported from all customer domains under the same Cloudflare account.
+To serve images through custom domains, an image url should be adjusted to the following format:
```
Comment (marciocloudflare) [inline products/images/src/content/cloudflare-images/serve-images/index.md]:
```suggestion
Image delivery is supported from all customer domains under the same Cloudflare account. To serve images through custom domains, an image URL should be adjusted to the following format:
```

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 5]
```diff
@@ -33,4 +33,27 @@ In this example:
...
+
+```txt
+https://<customerdomain.net>/cdn-cgi/imagedelivery/:images_account_hash/:image_id/:variant_name
+```
+
+Here is aforementioned example, which the change of using custom domain:
```
Comment (marciocloudflare) [inline products/images/src/content/cloudflare-images/serve-images/index.md]:
```suggestion
Example which a custom domain:
```

File: products/images/src/content/cloudflare-images/serve-images/index.md [hunk 6]
```diff
@@ -33,4 +33,27 @@ In this example:
...
+https://somecustomdomain.com/cdn-cgi/imagedelivery/ZWd9g1K7eljCn_KDTu_MWA/083eb7b2-5392-4565-b69e-aff66acddd00/public
+```
+
+In this example, `:images_account_hash`, `:image_id` and `:variant_name` are the same, but the hostname and prefix path is different: 
+
+- `somecustomdomain.com` custom domain.
+- `/cdn-cgi/imagedelivery` path to trigger cdn-cgi image proxy.
+- `ZWd9g1K7eljCn_KDTu_MWA` is the Images account hash.
+- `083eb7b2-5392-4565-b69e-aff66acddd00` is the image id.
+- `public` is the variant name.
```
Comment (marciocloudflare) [inline products/images/src/content/cloudflare-images/serve-images/index.md]:
```suggestion
- `/cdn-cgi/imagedelivery`: Path to trigger cdn-cgi image proxy.
- `ZWd9g1K7eljCn_KDTu_MWA:` The Images account hash.
- `083eb7b2-5392-4565-b69e-aff66acddd00`: The image id.
- `public`: The variant name.
```

**Review by marciocloudflare — CHANGES_REQUESTED** [review]
Could you take a look at my suggestions? Thanks!

**Review by sejoker — APPROVED** [review]

**Review by sejoker — APPROVED** [review]

**Review by marciocloudflare — APPROVED** [review]

**Pull request merged**